src/sidebar/reducers/moderation.js

```diff
@@ -48,11 +48,41 @@ var update = {
   FETCHED_HIDDEN_IDS: function (state, action) {
     return { hiddenByModerator: toSet(action.ids) };
   },
+
+  UPDATE_FLAG_STATUS: function (state, action) {
```

Comment:
Since this action operates on the `annotations` state, it should go in `reducers/annotations.js` along with the other actions that operate on the same state. Same applies to the tests.

Also, I've got a PR open which removes the `moderation` state key entirely.